lib/main.dart

```diff
@@ -47,11 +48,17 @@ Future _initSdks() async {
   ]);
 }
 
-Future _initFirebaseSdks() async => Firebase.initializeApp(
-      options: DefaultFirebaseOptions.currentPlatform(
-        await Config.getEnvFromBundleId(),
-      ),
-    );
+Future _initFirebaseSdks() async {
+  await Firebase.initializeApp(
+    options: DefaultFirebaseOptions.currentPlatform(
+      await Config.getEnvFromBundleId(),
+    ),
+  );
+  unawaited(
```

Comment:
Disable automatic collection, if it's not configured properly it can track data that is not valid 
`await _firebaseAnalytics.app.setAutomaticDataCollectionEnabled(false);`